extensions/objects/templates/filepath-editor.component.spec.ts

```diff
@@ -33,51 +33,84 @@ describe('File path editor', () => {
     component = TestBed.createComponent(
       FilepathEditorComponent).componentInstance;
   });
+
   it('should show svg editor when filepath represents an svg', () => {
     expect(component.imageEditorIsShown).toBeFalse();
     expect(component.svgEditorIsShown).toBeFalse();
+
     component.value = '/path/to/svg-image.svg';
     component.ngOnInit();
+
     expect(component.imageEditorIsShown).toBeFalse();
     expect(component.svgEditorIsShown).toBeTrue();
   });
+
   it('should show image editor when filepath does not represent an svg', () => {
     expect(component.imageEditorIsShown).toBeFalse();
     expect(component.svgEditorIsShown).toBeFalse();
+
     component.value = '/path/to/png-image.png';
     component.ngOnInit();
+
     expect(component.imageEditorIsShown).toBeTrue();
     expect(component.svgEditorIsShown).toBeFalse();
   });
+
   it('should cause the form to be invalid when filepath is empty', () => {
     spyOn(component.validityChange, 'emit');
+
     component.value = '';
     component.ngOnInit();
+
     expect(component.validityChange.emit).toHaveBeenCalledWith(
       { empty: false });
   });
+
   it('should detect change in value', () => {
     spyOn(component.valueChanged, 'emit');
+
     component.valueHasChanged('new value');
+
     expect(component.valueChanged.emit).toHaveBeenCalledWith('new value');
   });
+
   it('should detect change in validity', () => {
     spyOn(component.validityChange, 'emit');
+
     component.validityHasChanged({ empty: false });
+
     expect(component.validityChange.emit).toHaveBeenCalledWith(
       { empty: false });
   });
+
+  it('should reset the editor', () => {
```

Comment:
Can you please add when this function gets executed?

Reply:
Done.